index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
+			const retryAfter = options !== undefined &&
+				['number', 'string'].includes(typeof options.retryAfter) ?
+				Number(options.retryAfter) :
+				2000;
+
+			setTimeout(() => {
+				try {
+					originalKill(0);
```

Comment:
This is not needed, as `process.kill('SIGKILL')` is a noop if process is already killed.
See https://github.com/nodejs/node/blob/f5e2ad784ff8d51db275a7268660fc72cf58289d/lib/internal/child_process.js#L435

Reply:
Yes, good point!
46ee043aa907ef050c0d5eda4d0dc8ff31600cca